### PR TITLE
(ruka) add pgSphere to cnpg v1.19.0

### PR DESCRIPTION
**Pull request by cbarria**
- This is an upgrade for the cnpg from 1.17.0 (Chart 0.15.0) to 1.19.0 (Chart 0.17.0)
- same version of postgreSQL 14.5 but with pg_sphere extension 1.2.0 (requested by SLAC)
- pg_bouncer was added between the cnpg w/r service and the loadBalancer (max client 500 and pool size 100)
- backups to the s3 bucket are working great with same schedule (9am - 6pm CLT)
- Restore from cloud works perfect.
- Documentation was updated to reflect current use/status.

### CODE REVIEW

File: ruka/cnpg/cnpg-scheduledbackups.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   name: cnpg-backup-scheduled
   namespace: cloudnativepg
 spec:
-  schedule: "0 12,21 * * * *" #8AM CLT - 5PM CLT (after nightwork and daywork at the summit)
+  schedule: "0 0 12,21 * * *" #9AM CLT - 6PM CLT (after nightwork and daywork at the summit)
```
Comment (csilva-cl) [inline ruka/cnpg/cnpg-scheduledbackups.yaml:8]: This is UTC time, it will not work at specified CLT during CLST so better labeling as UTC

**Review by csilva-cl — CHANGES_REQUESTED** [review]

File: ruka/cnpg/README.md [hunk 2]
```diff
@@ -52,4 +53,4 @@ There is a yaml file provided for restoring the database from the S3
 
 fill the correct information inside the file ```cnpg-recovery.yaml``` (buckets directory)
 replace ```kubectl apply -f deploy.yaml``` for ```kubectl apply -f cnpg-recovery.yaml```
-inside ```cnpg.sh``` and then execute the script to start the cluster in recovery mode with previous backup.
+inside ```cnpgv2.sh``` and then execute the script to start the cluster in recovery mode with previous backup.
```
[the remark this answers is not in the document]
Comment (cbarria) [reply ruka/cnpg/README.md:56]: you are right, i did it to quickly avoid confusion with the previous deployment, so just by looking at it, you can discard which deployment it is, the old one, or the new one.
i will get the name back to cnpg.sh

File: ruka/cnpg/cnpg-scheduledbackups.yaml (hunk 1 above)
Comment (cbarria) [reply ruka/cnpg/cnpg-scheduledbackups.yaml:8]: its really both, the comment is to quickly check that the time on the cronjob (12 and 21) correspond to 9am-6pm CLT, but i can leave the time in UTC on the comment also

File: ruka/cnpg/cnpgv2.sh [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+  monitoring:
+    enablePodMonitor: true
+END
+#this needs to be changed to apply cnpg-recovery.yaml for recovery on existing cluster, 
+#refer to the file because s3 backup folder needs to be changed.
+kubectl apply -f cnpg-recovery.yaml
+
+#pgBouncer exposed with loadBalancer
+cat > pgbouncer-loadbalancer.yaml << END
```
[the remark this answers is not in the document]
Comment (cbarria) [reply ruka/cnpg/cnpgv2.sh:119]: going to move the config to a yaml file since it doesnt have environment variables

**Review by csilva-cl — APPROVED** [review]
looks good :)

**Pull request merged**